files/chrono/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "chrono",
+  "repo": "wanasit/chrono",
+  "files": {
+    "include": ["chrono.js", "chrono.min.js", "chrono.map.json"]
```

Comment:
Sorry. Added `chrono.map.json` in v1.1.0.
https://github.com/wanasit/chrono/tree/v1.1.0